gilded-rose-llc-frontend/__tests__/index.tests.tsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from "@testing-library/react";
+import Home from "../pages/index";
+
+describe("Home", () => {
+  it("should pass this test meaning that the testing framework is working", () => {
+    render(<Home />);
+
+    screen.debug();
```

Comment:
Ensure to take this out as it's similar to leaving a `console.log()` in your codebase.

Reply:
Do we need to have CI setup for the front end?

Reply:
@mrfishball I am unsure if we have enough time budgeted for this sprint to add CI. I would like to get CI working but our original estimate didn't account for CI.

If you'd like to see it added would it be ok for us to add it to our estimate for our next story?

Reply:
I see. I'm fine with setting up CI being a separate ticket/PR. I want to make sure that there's a plan to add CI. Thanks!

Reply:
Card #28 (Add CI for frontend) added to the backlog! cc. @jonmanahan